server/middleware/setupRequestLink.ts

```diff
@@ -0,0 +1,28 @@
+import express, { Router } from 'express'
+import session from 'express-session'
+import flash from 'connect-flash'
+import RequestLinkController from '../routes/link/RequestLinkController'
+import config from '../config'
+
+export default function setUpRequestLink(): Router {
+  const router = express.Router()
+  const requestLinksController = new RequestLinkController()
+
+  // TODO This uses the same max session age and secret as for a logged in user. Consider separating these into different env vars.
+  router.use(
+    session({
+      cookie: { maxAge: config.session.expiryMinutes * 60000 },
+      store: new session.MemoryStore(),
```

Comment:
Can we make this a RedisStore?

Reply:
Yes, though I will probably need some pointers. Next PR ?

Reply:
Yeah next PR.